dictionaries/error.definition.json

```diff
@@ -23,6 +23,12 @@
 		"fr": "",
 		"ko": ""
 	},
+	"need_to_set_acct_password": {
+		"en": "You need to set your password to finish setting up your account. Please use the forgot password link below.",
+		"es": "",
```

Comment:
Just curious if you have plans for translations yet?  I understand you may still be waiting for approval from @fillup about using Google Translate.  In the meantime, I'll send you a link to the spreadsheet we're using to gather input from our local experts.  I'm thinking you could drop yours in there and Phillip can ping those individuals again.